api/src/main/java/org/openmrs/module/kenyaemr/reporting/cohort/definition/covid/PatientsEnrolledOnCovidCohortDefinition.java

```diff
@@ -28,8 +28,11 @@
 
     @ConfigurationProperty
     private Date endDate;
-
-    public PatientsEnrolledOnCovidCohortDefinition() {
+    
+    @ConfigurationProperty
+    private String county;
```

Comment:
I think these attributes didn't work. May be we should just remove them

Reply:
Hey Anto.. I have updated the code accordingly